src/components/ImageDetails/ReuseSurvey.vue

```diff
@@ -29,6 +27,18 @@ export default {
   name: 'ReuseSurvey',
   components: { VLink },
   props: ['image'],
+  setup(props) {
+    const usageDataStore = useUsageDataStore()
+    const onReuseSurveyClick = async () => {
+      await usageDataStore.sendDetailPageEvent({
+        eventType: DETAIL_PAGE_EVENTS.REUSE_SURVEY,
+        resultUuid: props.image.id,
+      })
+    }
```

Comment:
Analytics requests shouldn't block; they don't have any bearing on the UI and blocking on them will make the UI less responsive.

```suggestion
    const onReuseSurveyClick = () => {
      usageDataStore.sendDetailPageEvent({
        eventType: DETAIL_PAGE_EVENTS.REUSE_SURVEY,
        resultUuid: props.image.id,
      })
    }
```

Reply:
We should add a comment about this so folks don't forget in the future.